src/NuGet.Core/NuGet.Protocol/Resources/ReadmeUriTemplateResource.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Protocol.Core.Types;
+using NuGet.Versioning;
+
+#if NETCOREAPP
+using System;
+#endif
+
+namespace NuGet.Protocol
+{
+    internal class ReadmeUriTemplateResource : INuGetResource
```

Comment:
Shouldn't this be part of the public API?
Perhaps this...
```suggestion
    public sealed class ReadmeUriTemplateResource : INuGetResource
```

Reply:
My thought was to keep it internal while this feature was in development and switch it to public once we decide we want to commit to supporting this implementation.

Reply:
Ok, but I was thinking if the resource is already added to nuget.org, and we're supporting it for nuget.org, we were past that decision point already.

Reply:
Yea, but it'll be easier to change our mind in the future about how it's implemented if we keep it internal.